skel/app.go

```diff
@@ -20,6 +20,7 @@ var App sonic.Runtime
 
 func serve() error {
 	App.Srv = gin.Default()
+	handlers.Version = Version
```

Comment:
Let's include the version to the sonic.Runtime
This will enforce the good practice of versioning applications.